test/unit/DCAStrategies/dca-strategies-positions-handler.spec.ts

```diff
@@ -169,6 +180,16 @@ contract('DCAStrategiesPositionsHandler', () => {
         expect(createCalls[0].permissionSets.length).to.be.equal(permissions.length);
         expect(createCalls[0].permissionSets).to.have.all.members(permissions);
       });
+      then('user position is saved correctly', async () => {
+        expect(userPosition.hub).to.be.equal(hub.address);
+        expect(userPosition.strategyId).to.be.equal(1);
+        expect(userPosition.strategyVersion).to.be.equal(1);
+
+        let expectedPositionsIds = [BigNumber.from(1), BigNumber.from(2)];
+        userPosition.positions.forEach((p, i) => {
```

Comment:
Let's not forget to check that `userPosition.positions.length === expectedPositionsIds.length`